lib/src/io/node.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2019 Google LLC
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:js/js.dart';
+
+export 'package:node_io/node_io.dart';
+
+// ignore: missing_js_lib_annotation
+@JS("process.stderr.write")
+external _writeToStderr(String text);
+
+class Stderr {
```

Comment:
This seems like it should be in `node_io`. If it's not there yet, can you send them a PR?

Reply:
I took a look at this, but implementing stderr properly there (rather than just what's needed here) is more of a yak shave than I'm up for at the moment, so I'll close this for now in favor of #62 to fix the compile error.